api/envoy/service/status/v2/csds.proto

```diff
@@ -41,15 +41,20 @@ enum ConfigStatus {
   // Management server has sent the config to client and received ACK.
   SYNCED = 1;
 
-  // Config is not sent.
+  // Config is not sent from the management server; or the client haven't
+  // received the requested config.
   NOT_SENT = 2;
 
   // Management server has sent the config to client but hasn’t received
   // ACK/NACK.
   STALE = 3;
 
-  // Management server has sent the config to client but received NACK.
+  // Management server has sent the config to client but received NACK; or the
+  // client has received the config but replied with NACK.
   ERROR = 4;
+
+  // The client has received the config and repied with ACK.
+  ACKED = 5;
```

Comment:
v2 is frozen, so requires justification for change here. Do you need this before we move to v3?

Reply:
I don't think we need this in v2.  Let's do it for v3 only.

Reply:
Changed in v2 reverted.